src/version/version.zig

```diff
@@ -14,9 +14,7 @@ pub const ClientVersion = struct {
 
     const Self = @This();
 
-    pub fn default() Self {
-        return CURRENT_CLIENT_VERSION;
-    }
+    pub const default = CURRENT_CLIENT_VERSION;
```

Comment:
would we want to change this too or is it ok as default?

Reply:
Actually I think `LATEST` or `CURRENT` would be better.

Reply:
Why even have a `default`? Why not use `CURRENT_CLIENT_VERSION` directly.

Reply:
I think it would be nice to be able to write `version = .CURRENT` instead of `sig.version.CURRENT_CLIENT_VERSION` - but I'm fine if we want to settle on just removing this one.

Reply:
that makes sense, i'm happy with `CURRENT`